echidna/core/spectra.py

```diff
@@ -188,73 +438,111 @@ def scale(self, num_decays):
         self._data = numpy.multiply(self._data, num_decays / self._num_decays)
         self._num_decays = num_decays
 
-    def shrink(self, energy_low=None, energy_high=None, radial_low=None,
-               radial_high=None, time_low=None, time_high=None):
-        """ Shrink the data such that it only contains values between energy_low
-        and energy_high (for example) by slicing. This updates the internal bin
+    def shrink(self, **kwargs):
+        """ Shrink the data such that it only contains values between low and
+        high for a given dimension by slicing. This updates the internal bin
         information as well as the data.
 
         Args:
-          energy_low (float): Optional new low bound of the energy.
-          energy_low (float): Optional new high bound of the energy.
-          radial_low (float): Optional new low bound of the radius.
-          radial_low (float): Optional new high bound of the radius.
-          time_low (float): Optional new low bound of the time.
-          time_low (float): Optional new high bound of the time.
-
-        Notes:
+          \**kwargs (float): Named parameters to slice on; note that these
```

Comment:
Ah ok I've realised that you were using the `\**` to try and display the `**` in `**kwargs`. This doesn't seem to be working very well - see screenshot.
![screenshot from 2015-09-07 13 49 14](https://cloud.githubusercontent.com/assets/1931666/9717367/1aacd800-556a-11e5-821f-5bd9cb7ad94c.png)
I think it would possibly be best to ignore the `**` - `kwargs` is the actual variable name anyway, `**` is the operator that unpacks the values from the dictionary (see http://stackoverflow.com/a/1769478).